app/flatpak-builtins-install.c

```diff
@@ -369,28 +372,20 @@ flatpak_builtin_install (int argc, char **argv, GCancellable *cancellable, GErro
                     continue;
                   else
                     {
-                      if (!flatpak_resolve_duplicate_remotes (dirs, this_remote, &dir_with_remote, cancellable, error))
-                        return FALSE;
-
-                      if (!opt_yes &&
-                          !flatpak_yes_no_prompt (TRUE, /* default to yes on Enter */
-                                                  _("Found refs similar to ‘%s’ in remote ‘%s’ (%s).\nIs that the remote you want to use?"),
-                                                  argv[1], this_remote, flatpak_dir_get_name (dir_with_remote)))
```

Comment:
I guess this makes the default-to-yes feature we added for this no longer used, but nevermind.

Reply:
It's still used in other prompts though